Client/wwwroot/index.html

```diff
@@ -40,7 +40,11 @@
 </head>
 
 <body>
-    <div id="app">Loading...</div>
+    <div id="app">
+        <div class="home-splash">
+            <img src="/img/devnxt-vertical-logo-black.png" alt="DevNxt Logo" width="50%">
```

Comment:
The case of this path doesn't match the image (should be "DevNxt-..." ).  This would only matter if we can guarantee the server is non-case sensitive (e.g. windows).
I run linux and so this failed for me.
You might want to check for oter cases of this.

Reply:
Yeah, we're deploying to an Azure site that's running a containerised linux distro, so while I'm sure it'll work, I'll make a future change to ensure everything is lowercase.